azkaban-webserver/src/main/less/flow.less

```diff
@@ -340,3 +340,17 @@ li.tree-list-item {
     }
   }
 }
+
+#analyzerButton {
```

Comment:
I wondered why your button needs to be different enough from other buttons that you had to define your own style. The more styles, the more work it will require to maintain. If there is a good reason to be different, it is fine. Please consider adding comments to explain.